java-repo-tools

```diff
@@ -0,0 +1 @@
+Subproject commit 5c9a3169e1ee8fd471c0fd7501b50e1b2bc16773
```

Comment:
Sounds good. I can do that. I also think while we're in the business of auto-updating Maven plugins with Jenkins, we can consider auto-updating submodule commit numbers, too.